Fixes Reverse IP problem

divshacker opened the pull request:
<!--
⚠️ Your pull request title should be short detailed and understandable for all.
⚠️ Increment the version number in the /VERSION file. If this is a bugfix, increment the patch version (the rightmost number), for example 0.18.2 becomes 0.18.3. If it's a new feature, increment the minor version (the middle number), for example 0.18.2 becomes 0.19.0. 
⚠️ Add a note to /CHANGELOG.md with your version number and the date, summarizing the changes.
⚠️ If your pull request fixes an open issue, please link to the issue.
⚠️ Rebase the latest changes from the develop branch. Assuming origin points to https://github.com/AgnostiqHQ/covalent, you should run git rebase origin/develop.
--> Hey I have commited all the changes as suggested by @FyzHsn in issue #202 
This PR fixes #202 

- [ ] I have added the tests to cover my changes.
- [ ] I have updated the documentation and CHANGELOG accordingly.
-  [X] I have read the CONTRIBUTING document.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/AgnostiqHQ/covalent?pullRequest=556) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/AgnostiqHQ/covalent?pullRequest=556) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/AgnostiqHQ/covalent?pullRequest=556) it.</sub>

divshacker commented:
There was a text written in which 0.0.0.0 was there. I left it unchanged as it seems it was warning for user. 
Apart from that I have changed and replaced pretty much every `0.0.0.0` to `localhost`. 
Please review this and let me know if there is any further thing needed. I haven't tested it locally. 
Thanks

divshacker commented:
Hey @scottwn May I know why there is conflicting issues? 
Also please review my PR and let me know further changes needed.
Thanks

venkatBala commented:
> Hey @scottwn May I know why there is conflicting issues? Also please review my PR and let me know further changes needed. Thanks

Hi @divshacker , thank you for your work on this issue. Could you kindly re-base your branch against `develop` and update the PR?

divshacker commented:
Hey @venkatBala I have already comparing my branch with `develop`. I don't know how to rebase it. Can you help me out please?
Thanks

scottwn commented:
@divshacker we have recently made some significant changes to the `develop` branch. Since your branch was based on `develop` prior to these changes, it contains hundreds of commits that are not in `develop`. 

What you'll need to do is remove the commits from this branch that aren't in `develop` and aren't relevant to the feature work. There are a few ways to do this, but I recommend an [interactive rebase](https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d):
1. Pull the latest changes from `develop`
2. While on your branch, `git rebase -i develop`
3. `drop` every commit except the ones you authored

Another approach would be to pull the changes from `develop`, make a new branch, apply your changes to that branch, and then, from this `localhost-fix` branch, `git reset --hard my-new-branch`.
